tools/files/linters.yml

```diff
@@ -0,0 +1,18 @@
+name: Linters
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  linters:
+    uses: ansible-network/github_actions/.github/workflows/tox-linters.yml@main
+  ansible-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Run ansible-lint
+        uses: ansible/ansible-lint@v6.21.0
```

Comment:
What happens if you pass the versioning of ansible-lint as a parameter? Since versioning changes frequently, it is easier to pass it as a parameter than to change it manually.

Reply:
Added it to template.